CHANGELOG.md

```diff
@@ -1,2 +1,13 @@
 ## Unreleased
 
+## v0.9.1
+
+IMPROVEMENTS:
+* Updated dependencies:
+   * `github.com/couchbase/gocb/v2` v2.3.3 -> v2.6.3
+   * `github.com/hashicorp/go-hclog` v1.0.0 -> v1.5.0
+   * `github.com/hashicorp/go-version` v1.3.0 -> v1.6.0
+   * `github.com/hashicorp/vault/sdk` v0.5.3 -> v0.9.0
+   * `github.com/mitchellh/mapstructure` v1.5.0 -> v1.5.0
```

Comment:
Looks like there's a small bug in the python script for deps that moved from indirect to direct?

Reply:
Ah, good catch. I'll fix that. Thanks.